client/src/pages/layouts/components/ProfileAvatar.jsx

```diff
@@ -1,9 +1,47 @@
-import {Avatar} from '@material-ui/core';
-import React from 'react';
-import sampleAvatar from '../../static/sampleAvatar.png';
+import {Avatar, Button} from '@material-ui/core';
+import React, {useState} from 'react';
+import UserContext from '../../../contexts';
+import UserMenu from './UserMenu';
 
 const ProfileAvatar = () => {
-  return <Avatar alt="UserName" src={sampleAvatar} />;
+  const [anchorElem, setAnchorElem] = useState(null);
+  const [avatar, setAvatar] = useState('');
+
+  const handleClickOpen = (event) => {
+    setAnchorElem(event.currentTarget);
+  };
+
+  return (
+    <UserContext.Consumer>
+      {(value) => {
+        if (value.user) {
+          if (avatar === '') {
+            setAvatar(value.user.avatar);
+          }
+          return (
+            <div>
+              <Button onClick={handleClickOpen}>
+                <Avatar
+                  alt={value.user.name}
+                  src={avatar || value.user.avatar}
+                />
+              </Button>
+
```

Comment:
Hey I uploaded the avatar photo successfully, and I see the avatar key entry in my user model as well. After I re-login, the profile picture is loaded with the default photo, not my avatar photo.

Reply:
Just fixed the issue too, turns out the avatar field doesnt get passed on initial user-login, but handled that issue 👍